The job **`x86_64-gnu-llvm-17`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9464116689/job/26070750985) [(plain)](https://github.com/rust-lang/rust/commit/8985a5494dd3fa83599e9ebc1eac90dde1b0b04c/checks/26070750985/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
#16 exporting to docker image format
#16 sending tarball 29.3s done
#16 DONE 34.7s
##[endgroup]
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-17]
---
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-17', '--enable-llvm-link-shared', '--set', 'rust.thin-lto-import-instr-limit=10', '--set', 'change-id=99999999', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-17/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.thin-lto-import-instr-limit := 10
configure: change-id            := 99999999
---
   Compiling addr2line v0.22.0
error[E0753]: expected outer doc comment
  --> library/std/src/alloc.rs:76:1
   |
76 | //! This type implements the [`GlobalAlloc`] trait. Currently the default
   |
   |
   = note: inner doc comments like this (starting with `//!` or `/*!`) can only appear before items
help: you might have meant to write a regular comment
   |
76 - //! This type implements the [`GlobalAlloc`] trait. Currently the default
76 + // This type implements the [`GlobalAlloc`] trait. Currently the default

error[E0753]: expected outer doc comment
  --> library/std/src/alloc.rs:77:1
   |
   |
77 | //! global allocator is unspecified. Libraries, however, like `cdylib`s and
   |
   |
   = note: inner doc comments like this (starting with `//!` or `/*!`) can only appear before items
help: you might have meant to write a regular comment
   |
77 - //! global allocator is unspecified. Libraries, however, like `cdylib`s and
77 + // global allocator is unspecified. Libraries, however, like `cdylib`s and

error[E0753]: expected outer doc comment
  --> library/std/src/alloc.rs:78:1
   |
   |
78 | //! `staticlib`s are guaranteed to use the [`System`] by default and as such
   |
   |
   = note: inner doc comments like this (starting with `//!` or `/*!`) can only appear before items
help: you might have meant to write a regular comment
   |
78 - //! `staticlib`s are guaranteed to use the [`System`] by default and as such
78 + // `staticlib`s are guaranteed to use the [`System`] by default and as such

For more information about this error, try `rustc --explain E0753`.
error: could not compile `std` (lib) due to 3 previous errors
Build completed unsuccessfully in 0:00:16
```

</details>

--- a/library/std/src/alloc.rs
+++ b/library/std/src/alloc.rs
@@ -73,7 +73,9 @@ pub use alloc_crate::alloc::*;
 /// work, such as to serve alignment requests greater than the alignment
 /// provided directly by the backing system allocator.
 ///
-/// This type implements the `GlobalAlloc` trait and Rust programs by default
+/// This type implements the [`GlobalAlloc`] trait. Currently the default
+/// global allocator is unspecified. Libraries, however, like `cdylib`s and
+/// `staticlib`s are guaranteed to use the [`System`] by default and as such
 /// work as if they had this definition:
 ///
 /// ```rust

